packages/react-app/src/components/modals/ClaimTransferModal.jsx

```diff
@@ -99,11 +99,25 @@ export const ClaimTransferModal = () => {
     } else if (claimable) {
       try {
         setClaiming(true);
-        await executeSignatures(ethersProvider, foreignAmbAddress, message);
+        const { error, alreadyClaimed, data } = await executeSignatures(
+          ethersProvider,
+          foreignAmbAddress,
+          message,
+        );
         setLoadingText('Waiting for Execution');
+        if (error) {
+          throw error;
+        }
+
+        if (!data && alreadyClaimed) {
+          showError(
+            `The transfer was already executed. Check your balance of this token in ${getNetworkName(
+              foreignChainId,
+            )}</strong>`,
```

Comment:
is `</strong>` a typo here ? Also we might wanna change the error message in line 85 as well to match this error message.
also please call `setExecuted(true)` as well here.